classes/02class/exercises/c02-network04/alanlima/README.md

```diff
@@ -0,0 +1,204 @@
+# c02-network04
+
+## Commands Execution Output
+
+- Commands for creating the subnets:
+```
+# Create VPC
+
+$ aws ec2 create-vpc \
+    --cidr-block 172.16.0.0/16 \
+    --no-amazon-provided-ipv6-cidr-block \
+    --instance-tenancy default
```

Comment:
isn't your VPC created already in lab03?

Reply:
I usually delete all resources once I finished the lab. so I needed to create it again. 😅😅